packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -396,6 +397,17 @@ class ApplicationCommand extends Base {
     }
     return true;
   }
+  
+  /**
+   * When concatenated with a string, this automatically returns the command's mention instead of the ApplicationCommand object.
+   * @returns {string}
+   * @example
+   * // Logs: Command: </ping:123456789012345678>
+   * console.log(`Command: ${command}`);
+   */
+  toString() {
+    return chatInputApplicationCommandMention(`${this.name}`, this.id);
```

Comment:
```suggestion
    return chatInputApplicationCommandMention(this.name, this.id);
```
Why do you convert the name string to a string?

Reply:
![image](https://user-images.githubusercontent.com/75100939/200318243-774930ec-6e71-4b27-b40a-806450ced718.png)
TS was complaining about the typings, so I wasn't sure what the best practice was here since the documentation clearly shows that the name is always present.  Is it best to just ignore TS here?

Reply:
https://github.com/discordjs/discord.js/pull/8818#discussion_r1015402809